ground/gcs/src/libs/utils/pathutils.cpp

```diff
@@ -33,13 +33,21 @@
 
 namespace Utils {
 
-QString PathUtils::settingsFilename = "";
+PathUtils * Utils::PathUtils::instance = 0;
```

Comment:
I think this should probably change to NULL.  0 used to be favored for pointers in C++, but with the emergence of C++11 and nullptr/nullptr_t NULL is safer in various ways.